src/AutoRest.CSharp/Common/AutoRest/Plugins/Configuration.cs

```diff
@@ -108,6 +112,23 @@ private static bool GetOptionValue(IPluginCommunication autoRest, string option)
             }
         }
 
+        private static string GetOptionStringValue(IPluginCommunication autoRest, string option, Func<string, string>? func)
```

Comment:
Do we need a new method?  Can we use `GetOptionValue(autoRest, Options.ModelNamespace) ?? "default value"` ?

Reply:
GetOptionValue returns a bool, it was originally written for bool options only. I did not see a good way to to use it for string options, so I added a new method. And since the string may need extra processing, I added a new signature of `Func<string, string>? func` parameter.